project-base/storefront/pages/categories/[categorySlug].tsx

```diff
@@ -129,6 +134,20 @@ export const getServerSideProps = getServerSidePropsWrapper(
                 context,
                 client,
                 ssrExchange,
+                prefetchedQueries: [
+                    {
+                        query: AdvertsQueryDocument,
+                        variables: { positionName: 'productList', categoryUuid },
+                    },
+                    {
+                        query: AdvertsQueryDocument,
+                        variables: { positionName: 'productListMiddle', categoryUuid },
+                    },
+                    {
+                        query: AdvertsQueryDocument,
+                        variables: { positionName: 'productListSecondRow', categoryUuid },
```

Comment:
The behavior of the advert in category has changed. 

Previously (= on the 15.0 branch), when no category was set, the advert was not rendered anywhere. Now its rendered in the each category

Reply:
Yes, I have found that the behaviour changed because of one specific check was previously done on the FE and aparently the BE does not check for that:

(content of the filter function `shouldBeShown` in `Adverts.tsx`)
```js
...
const isAdvertWithCategoryRestriction = ['productList', 'productListMiddle', 'productListSecondRow'].includes(
    advert.positionName,
);

if (isAdvertWithCategoryRestriction && !advert.categories.length) {
    return false;
}
...
```
So if the categories array for these 3 positionNames `productList`, `productListMiddle` and `productListSecondRow` is empty, the advert is not shown.